TARGETS

```diff
@@ -71,9 +73,14 @@ default_allocator = read_config("fbcode", "default_allocator")
 
 sanitizer = read_config("fbcode", "sanitizer")
 
+use_jemalloc = ((is_opt_mode or is_dbg_mode)
+                and default_allocator.startswith("jemalloc")
```

Comment:
As per @anand1976 's comment [here](https://github.com/facebook/rocksdb/pull/4747/files#r239172915), are you expecting dev mode builds to still have issues?

Reply:
yes, the issue is with dev-nonsan mode. It seems default_allocator is jemalloc but jemalloc is not linked with the binary in this mode. I'm asking in buck Q&A but before we have a proper solution, I'm working around for opt and dbg modes.